minari/data_collector/data_collector.py

```diff
@@ -250,8 +265,8 @@ def reset(
         assert STEP_DATA_KEYS.issubset(step_data.keys())
 
         # If last episode in global buffer has saved steps, we need to check if it was truncated or terminated
-        # If not, then we need to auto-truncate the episode
-        if len(self._buffer[-1]["actions"]) > 0:
+        # If not (empty dicitionary), then we need to auto-truncate the episode.
```

Comment:
comments should ideally be more understandable than this, I'd rephrase to "if the dictionary is not empty, "

Reply:
Rephrased.